spec/graphql/argument_spec.rb

```diff
@@ -15,7 +15,7 @@
     }
 
     expected_error = %|Query is invalid: field "invalid" argument "invalid" default value ["123"] is not valid for type Float|
-    assert_equal expected_error, err.message
+    assert_includes err.message, expected_error
```

Comment:
I modified the message to include the exception information, if there was one. In this case, the exception shows the undefined method `to_f` for array. I thought it would be helpful for debugging if you have a custom scalar, and the default value is being rejected.

This code could assert equality, in newer versions of Ruby, the messaging for undefined method exceptions changed:

Ruby 2.3.1:
```
2.3.0 :001 > ['abc'].to_f
NoMethodError: undefined method `to_f' for ["abc"]:Array
Did you mean?  to_s
               to_a
               to_h
	from (irb):1
	from /Users/theorygeek/.rvm/rubies/ruby-2.3.0/bin/irb:11:in `<main>'
```

Ruby 2.2.3:
```
2.2.3 :001 > ['abc'].to_f
NoMethodError: undefined method `to_f' for ["abc"]:Array
	from (irb):1
	from /Users/theorygeek/.rvm/rubies/ruby-2.2.3/bin/irb:11:in `<main>'
```